operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -217,6 +217,19 @@ spec:
                     description: loadBalancer holds parameters for the load balancer.
                       Present only if type is LoadBalancerService.
                     properties:
+                      allowedSourceRanges:
+                        description: allowedSourceRanges specifies a list of IP address
```

Comment:
I would use the word "whitelist", "allowlist", or "safelist", somewhere in the description, because those are the words I'd be looking for if I were trying to figure out how to configure this option.

Reply:
Good idea! I used allowlist in the description.